packages/mui-base/src/TabsUnstyled/TabsUnstyledProps.ts

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { OverrideProps } from '@mui/types';
+
+export interface TabsOwnerState
+  extends Omit<TabsUnstyledProps, 'component' | 'components' | 'componentsProps'> {
+  hidden: boolean;
+}
+
+export interface TabsUnstyledOwnProps {
+  /**
+   * The content of the component.
+   */
+  children?: React.ReactNode;
+  /**
+   * The value of the currently selected `Tab`.
+   * If you don't want any selected `Tab`, you can set this prop to `false`.
+   */
+  value?: string | number | false;
+  /**
+   * The default value. Use when the component is not controlled.
+   */
+  defaultValue?: string | number | false;
+  /**
+   * The component orientation (layout flow direction).
+   * @default 'horizontal'
+   */
+  orientation?: 'horizontal' | 'vertical';
+  /**
+   * The direction of the text.
+   * @default 'ltr'
+   */
+  direction?: 'ltr' | 'rtl';
+  className?: string;
+  /**
+   * The components used for each slot inside the Tabs.
+   * Either a string to use a HTML element or a component.
+   * @default {}
+   */
+  components?: {
+    Root?: React.ElementType;
+  };
+  /**
+   * The props used for each slot inside the Tabs.
+   * @default {}
+   */
+  componentsProps?: {
+    root?: React.ComponentPropsWithRef<'div'> & { ownerState: TabsOwnerState };
```

Comment:
Add `& TabsUnstyledComponentsPropsOverrides`, similarly to how it's done in other base components.

(+ other files)

Reply:
Good catch! Done.